### PR TITLE
Fixed an issue wher the tab title was missing in the documents library

### LINKED ISSUE #8091: No title in tab when in "Documents" library
### Description

When navigating to the "Documents" library from within the link in the sidebar, the tab title appears blank. This behavior does not occur for Music, Pictures, and Videos, and does not occur when opening "Documents" from the "Home" tab. It appears to only occur when accessing the page from the sidebar.

### Steps To Reproduce

Open the "Documents" library from the sidebar and check the tab title.

### Expected behavior

When selecting the "Documents" library from the sidebar, the tab should have a title.

### Files Version

Version: 2.1.5.0 OS Version: 10.0.22000.434

### Windows Version

Windows 11 21H2 22000.434

### Relevant Assets/Logs

Preview of the tab area with all libraries opened from the sidebar (In order: Documents, Music, Pictures, Videos)

image

**Cannot pinpoint when this bug first started, but it was also noticed in Version 2.1.3 before updating to the latest version.**

### PR DESCRIPTION
**Pull request by puppetsw**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #8091

**Details of Changes**
Add details of changes here.
- Added missing resource string for 'SidebarDocuments'

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

File: src/Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -2744,4 +2744,7 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="CloseOtherTabs" xml:space="preserve">
     <value>Close other tabs</value>
   </data>
-</root>
+  <data name="SidebarDocuments" xml:space="preserve">
```
Comment (yaichenbaum) [inline src/Files/Strings/en-US/Resources.resw:2747]:
```suggestion
  <data name="Documents" xml:space="preserve">
```
Can you edit the code to look for `Documents`?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]
LGTM

File: src/Files/Strings/en-US/Resources.resw (hunk 1 above)
Comment (puppetsw) [reply src/Files/Strings/en-US/Resources.resw]:
```cs
switch (libName)
{
	case "Documents":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	case "Pictures":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	case "Music":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	case "Videos":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	default:
		tabLocationHeader = library.Text; // Show original name
		break;
}
```
@yair100 This is where it's getting the SidebarDocuments string from. Did you want me to go ahead and change the others as well to keep it consistent?

Comment (yaichenbaum) [reply src/Files/Strings/en-US/Resources.resw:2747]: That's a good idea, thank you!

File: src/Files/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -2744,4 +2744,16 @@ We use App Center to track which settings are being used, find bugs, and fix cra
...
```
Comment (yaichenbaum) [inline src/Files/Strings/en-US/Resources.resw:2747]:
```suggestion
  <data name="Documents" xml:space="preserve">
```

**Comment by yaichenbaum** [comment]
@puppetsw can you remove the strings that aren't being used anymore?

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**